packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -362,7 +362,7 @@ X-Okta-Request-Id: reqVy8wsvmBQN27h4soUE3ZEnA
 
 ### Cross-Origin Resource Sharing (CORS)
 
-[Cross-Origin Resource Sharing (CORS)](http://en.wikipedia.org/wiki/Cross-Origin_Resource_Sharing) is a mechanism that allows a web page to make an AJAX call by using [XMLHttpRequest (XHR)](http://en.wikipedia.org/wiki/XMLHttpRequest) to a domain that is different from the one where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers, in accordance with the [same origin security policy](http://en.wikipedia.org/wiki/Same_origin_policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether to allow the cross-origin request.
+[Cross-Origin Resource Sharing (CORS)](https://fetch.spec.whatwg.org/#cors-protocol) is a mechanism that allows a web page to make an AJAX call by using [XMLHttpRequest (XHR)](https://xhr.spec.whatwg.org/) to a domain that is different from the one where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers, in accordance with the [same origin security policy](https://www.w3.org/Security/wiki/Same_Origin_Policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether to allow the cross-origin request.
```

Comment:
```suggestion
[Cross-Origin Resource Sharing (CORS)](https://fetch.spec.whatwg.org/#cors-protocol) is a mechanism that allows a web page to make an AJAX call. The AJAX call uses [XMLHttpRequest (XHR)](https://xhr.spec.whatwg.org/) to a domain that's different from the one where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers, in accordance with the [same origin security policy](https://www.w3.org/Security/wiki/Same_Origin_Policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether to allow the cross-origin request.
```